src/rpc/governance.cpp

```diff
@@ -132,12 +132,13 @@ void gobject_prepare_help()
                 "2. revision      (numeric, required) object revision in the system\n"
                 "3. time          (numeric, required) time this object was created\n"
                 "4. data-hex      (string, required)  data in hex string form\n"
+                "5. use-IS        (boolean, optional, default=false) InstantSend lock the collateral, only requiring one chain confirmation\n"
                 );
 }
 
 UniValue gobject_prepare(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() != 5)
+    if (request.fHelp || (request.params.size() >= 5 && request.params.size() <= 6)) 
```

Comment:
This won't work, this says "if you provided 5 or 6 params, here is the help message" but 5 or 6 params is the correct number of params.

```suggestion
    if (request.fHelp || (request.params.size() != 5 && request.params.size() != 6)) 
```